exchanges/persisted-fetch/src/persistedFetchExchange.ts

```diff
@@ -81,11 +92,22 @@ export const persistedFetchExchange: Exchange = ({
               body.extensions = undefined;
               // Disable future persisted queries
               supportsPersistedQueries = false;
-              return makePersistedFetchSource(operation, body, dispatchDebug);
+              return makePersistedFetchSource(
+                operation,
+                body,
+                dispatchDebug,
+                false
+              );
             } else if (result.error && isPersistedMiss(result.error)) {
               // Add query to the body but leave SHA256 hash intact
               body.query = query;
-              return makePersistedFetchSource(operation, body, dispatchDebug);
+              return makePersistedFetchSource(
+                operation,
+                body,
+                dispatchDebug,
+                !!(options as PersistedFetchExchangeOptions)
```

Comment:
I believe this one should be set to `false` as well, since it sends the entire `query`